config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/fog-connection.lib.yml

```diff
@@ -0,0 +1,22 @@
+#@ load("@ytt:data","data")
+#@ load("@ytt:assert","assert")
+
+#@ def fog_connection():
```

Comment:
this `fog_connection` refactor seems like it could be a good PR candidate to capi-k8s-release to remove some duplicated configuration.

Reply:
Yup I figured that lived there, but I figured getting the initial PR into here for usability discussion made sense first.

Reply:
Another way to go might be to create the PR directly on capi-k8s-release but then "broadcast" the PR by dropping a note in #cf-for-k8s on CloudFoundry slack